stl/inc/random

```diff
@@ -508,6 +508,34 @@ _NODISCARD _Uint _Next_linear_congruential_value(_Uint _Prev) noexcept {
     }
 }
 
+// Optimized specialization for minstd_rand0
+template <>
+_NODISCARD unsigned int _Next_linear_congruential_value<unsigned int, 16807, 0, 2147483647>(
+    unsigned int _Prev) noexcept {
+    constexpr unsigned long long _All = 16807ULL;  

```

Comment:
You need to clang-format this file; see the failure logs in https://dev.azure.com/vclibs/STL/_build/results?buildId=18201&view=results . (We've configured clang-format to strip trailing whitespace, align `=`, and more.)

Reply:
Done